src/test/java/uk/gov/hmcts/dts/fact/services/admin/AdminAuditServiceTest.java

```diff
@@ -51,12 +52,20 @@ class AdminAuditServiceTest {
 
     @BeforeAll
     static void beforeAll() {
+        ZoneId utcZone = ZoneId.of("Europe/London");
+
         AUDIT_DATA.add(new Audit("Test String", new AuditType(1, "test"),
-                               "data before", "data after",
```

Comment:
It might be good to have one test where the utc date/time from repository is located within British Summer Time and assert that the service has changed it to BST

Reply:
added a bst one